internal/host/hostcommands/tang_connectivity_check_cmd_test.go

```diff
@@ -38,6 +39,58 @@ var _ = Describe("tangConnectivitycheckcmd", func() {
 
 	})
 
+	It("get_step: Tang imported cluster should use host ignition", func() {
+
+		const hostIgnition = `{
+			"ignition": {
+			  "config": {},
+			  "version": "3.2.0"
+			},
+			"storage": {
+				"luks": [
+					{
+					  "clevis": {
+						"tang": [
+						  {
+							"thumbprint": "nWW89qAs1hDPKiIcae-ey2cQmUk",
+							"url": "http://foo.bar"
+						  }
+						]
+					  },
+					  "device": "/dev/disk/by-partlabel/root",
+					  "name": "root",
+					  "options": [
+						"--cipher",
+						"aes-cbc-essiv:sha256"
+					  ],
+					  "wipeVolume": true
+					}
+				],
+			  "files": []
+			}
+		  }`
+
+		By("Set cluster to imported and inject host ignition for test", func() {
```

Comment:
Why are you not using the V2ImportCluster function to create the cluster?

Reply:
There is no precedent of V2 API's being used in the hostcommand tests and ultimately the only field that affects our test behaviour is the `imported` field, I think this approach is simpler.

I don't want to over complicate things and end up testing API calls instead of the code we are actually testing.